src/ts/utils/roam.ts

```diff
@@ -58,7 +63,80 @@ export const Roam = {
         await Keyboard.pressEnter(15);
         await Keyboard.pressEnter();
         document.execCommand('paste')
+    },
+
+    moveCursorToStart() {
+        this.applyToCurrent(
+            node =>
+                new RoamNode(
+                    node.text,
+                    // new Selection(node.text.length * 2, node.text.length * 2)
+                    new Selection(0,0)
+                ))
+    },
+
+    moveCursorToEnd() {
+        this.applyToCurrent(
+            node =>
+                new RoamNode(
+                    node.text,
+                    new Selection(node.text.length * 2, node.text.length * 2)
+                ))
+    },
+
+    writeText(text: string) {
```

Comment:
This potentially fails silently, I think it needs to provide indication of result

Reply:
this works now I guess?